lib/panes/output-area.js

```diff
@@ -15,6 +15,8 @@ export default class OutputPane {
   constructor(store: store) {
     this.element.classList.add("hydrogen");
 
+    atom.config.set("Hydrogen.outputAreaDefault", true);
```

Comment:
Why are you setting the config value here?

Reply:
With the last commit we don't need to. Originally it was so that if you toggle open the output pane the setting would turn on.

Reply:
👍

Reply:
Is there follow up that needs to be done here?